docs/pages/_app.js

```diff
@@ -57,7 +66,16 @@ function GestaltApp(
 GestaltApp.getInitialProps = async (appInitialProps: AppInitialProps): Promise<AppInitialProps> => {
   const initialProps = await NextApp.getInitialProps(appInitialProps);
   const cookieHeader = appInitialProps?.ctx?.req?.headers?.cookie;
-  return { ...initialProps, ...(cookieHeader ? { cookieHeader } : {}) };
+
+  // This should be replaced with a more sophisticated userAgent detection
+  // var ua = parser(appInitialProps?.ctx?.req?.headers['user-agent']);
+  // const isMobile = ua?.device?.type === 'mobile';
```

Comment:
I didn't impleemnt this because my yarn.lock gets messed up. I need to reset my repo... Someone should install the dependency so I can use this logic.